### PR TITLE
Create login security functional.

### PR DESCRIPTION
**Pull request by HotoMazur**
Create the login functional. 
Add 3 files:
-LoginConfig
-Security config
-AppBasicAuthenticationEntryPoint

To start work, build project to add two new implementation (security and web)
To log in to account go to localhost:8080 and use this data:

username = user
password = password

### CODE REVIEW

File: src/main/java/com/coherentsolutions/pot/insurance/LoginControl.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.coherentsolutions.pot.insurance;
```
Comment (SavaLX) [inline src/main/java/com/coherentsolutions/pot/insurance/LoginControl.java:1]: Please use the right distribution by directories(packages).
Please check this link https://climbtheladder.com/10-java-package-structure-best-practices/ to

File: src/main/java/com/coherentsolutions/pot/insurance/LoginControl.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class LoginControl {
```
Comment (SavaLX) [inline src/main/java/com/coherentsolutions/pot/insurance/LoginControl.java:7]: It's better to rename it to "LoginController"
Don't make class names shorter

File: src/main/java/com/coherentsolutions/pot/insurance/security/SecurityConfig.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                        .requestMatchers("/login").permitAll()
+                        .anyRequest().authenticated())
+                .httpBasic(Customizer.withDefaults())
+                .csrf(AbstractHttpConfigurer::disable);
+        return http.build();
+    }
+
+    @Bean
+    public InMemoryUserDetailsManager userDetailsService(){
+        UserDetails user = User
```
Comment (SavaLX) [inline src/main/java/com/coherentsolutions/pot/insurance/security/SecurityConfig.java:34]: Let's rename it to "admin" and pass "ADMIN_ROLE" to roles as we implementing the admin portal for now

**Comment by SavaLX** [comment]
For now just leave the name of branch as it is, but for future add jira ticket number to the name of branch and also to the initial commit.
At this moment when you will push your changes after resolving the comments, add jira ticket number to your commit.
And don't forget to delete your old PR

**Review by SavaLX — CHANGES_REQUESTED** [review]
Request changes

**Comment by HotoMazur** [comment]
@SavaLX Change all problem that you show. Send new commit with LJI-3 tag.

**Review by anddevdev — APPROVED** [review]

File: src/main/java/com/coherentsolutions/pot/insurance/login/LoginController.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package com.coherentsolutions.pot.insurance.login;
```
Comment (SavaLX) [inline src/main/java/com/coherentsolutions/pot/insurance/login/LoginController.java:1]: This is one possible strategy for grouping classes into packages (grouping by feature), but I would suggest using grouping by layer. A large application is easier to navigate when classes are grouped into layers. 
Example: all controllers must be in the "controller" package.

Comment (HotoMazur) [reply src/main/java/com/coherentsolutions/pot/insurance/login/LoginController.java]: Implement your task. Create controller package and relocate the LoginController to this package. Delete login package.

**Review by SavaLX — APPROVED** [review]

**Pull request merged**